jerry-main/main-unix.c

```diff
@@ -144,7 +144,7 @@ print_unhandled_exception (jerry_value_t error_value)
   }
   err_str_buf[err_str_size] = 0;
 
-  jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Script Error: unhandled exception: %s\n", err_str_buf);
+  jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Unhandled Exception: %s\n", err_str_buf);
```

Comment:
I'm not fully convinced about the necessity of this change. In jerry-main, all our diagnostics are prefixed: if something is not completely OK, we print something with "Warning: " prefix, if something went totally wrong in jerry-main from which we cannot recover, we use "Error: " prefix, and if some show-stopper issue comes from the script, we prefix it with "Script Error: ". This way it is easier to distinguish whether source of the error is in the tool or in the executed script.
(Note: the same "Script Error: " prefix is used in the JS assertion handler.)